website/docs/r/auditing.html.markdown

```diff
@@ -23,7 +23,7 @@ resource "mongodbatlas_auditing" "test" {
 
 ## Argument Reference
 
-* `project_id` - (Required) The unique ID for the project to configure auditing.
+* `project_id` - (Required) The unique ID for the project to configure auditing. **Note! When changing this value to a different project_id it will delete the current audit settings for the original project that was assigned to.**
```

Comment:
```suggestion
* `project_id` - (Required) The unique ID for the project to configure auditing. **Note: When changing this value to a different project_id it will delete the current audit settings for the original project that was assigned to.**
```